hooks/environment

```diff
@@ -56,7 +72,7 @@ retry() {
 }
 
 check_command vault
-
+check_command buildkite-agent
```

Comment:
This fails the script if not found - this _shouldn't_ ever happen, so happy to ignore if we're confident, but wanted to flag it may cause issues

Reply:
If there are cases where buildkite agent can not be present we can remove it. Is there such a case? I thought that buildkite agent is supposed to be present in any instance that gets created.

Reply:
No it should be there (it wasn't when I tested the script locally, cause I'm not a Buildkite agent :joy_cat:)